cmd/buildah/bud.go

```diff
@@ -23,9 +23,11 @@ type budResults struct {
 }
 
 func init() {
-	var (
-		budDescription = "\n  Builds an OCI image using instructions in one or more Dockerfiles."
-	)
+	budDescription := `
+  Builds an OCI image using instructions in one or more Dockerfiles.
+
+  If no arguments specified, it will assume the current working directory as
```

Comment:
Day late and a dollar short I am, but  s/arguments specified/arguments are specified/

Reply:
I fix this with the next PR.